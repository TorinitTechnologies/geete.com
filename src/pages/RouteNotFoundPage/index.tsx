import React from 'react';

const RouteNotFoundPage = () => {
	return <div data-testid={'page_not_found'}>RouteNotFoundPage</div>;
};

export default RouteNotFoundPage;
