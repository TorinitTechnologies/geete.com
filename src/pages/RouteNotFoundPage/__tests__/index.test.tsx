import { render, screen } from '@testing-library/react';
import RouteNotFoundPage from '..';

describe('Page Not Found Tests', () => {
	it('should render Page Not Found page', async () => {
		render(<RouteNotFoundPage />);
		const pageNotFoundElement = await screen.getByTestId('page_not_found');
		expect(pageNotFoundElement).toBeInTheDocument();
	});
});
