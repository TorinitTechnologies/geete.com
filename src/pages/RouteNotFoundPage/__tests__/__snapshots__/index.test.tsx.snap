// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Not Found Tests renders correctly 1`] = `
<div
  data-testid="page_not_found"
>
  RouteNotFoundPage
</div>
`;
